dash/dash.py

```diff
@@ -280,6 +280,11 @@ def __init__(
         elif isinstance(server, bool):
             name = name if name else "__main__"
             self.server = flask.Flask(name) if server else None
+            if self.server is not None:
+                # flask-compress==1.6.0 changed default to ['br', 'gzip']
+                # and non-overridable default compression with Brotli is
+                # causing performance issues
+                self.server.config["COMPRESS_ALGORITHM"] = ["gzip"]
```

Comment:
Hesitant to override the values for servers passed as parameter https://github.com/plotly/dash/pull/1426/files#diff-228c9aefac729977642672aa1416bacaR277

Reply:
Whenever we call `Compress(self.server)` we're effectively modifying the server's settings - and by default we do this even for user-provided servers. Perhaps we can check whether `server.config["COMPRESS_ALGORITHM"]` already has a value, and only set it if not?

Reply:
Turns out that running the code above against `1.5.0` fails and there is no `__version__` or `VERSION` on `flask_compress` so determining the version needs to be done through `pkg_resources`. Updating.

Reply:
https://github.com/plotly/dash/pull/1426#discussion_r500570299 - 👍